src/domain/app/App.tsx

```diff
@@ -69,6 +55,27 @@ const ApolloWrapper: React.FC<PropsWithChildren> = ({ children }) => {
 };
 
 const App: React.FC = () => {
+  const matomoTracker = useMemo(
+    () =>
+      new MatomoTracker({
+        urlBase: getValue(import.meta.env.REACT_APP_MATOMO_URL_BASE, ''),
+        siteId: import.meta.env.REACT_APP_MATOMO_SITE_ID,
+        srcUrl: import.meta.env.REACT_APP_MATOMO_SRC_URL,
+        enabled: import.meta.env.REACT_APP_MATOMO_ENABLED === 'true',
+        configurations: {
+          setDocumentTitle: `${document.location}${document.title}`,
+          setCookieDomain: import.meta.env.REACT_APP_MATOMO_COOKIE_DOMAIN,
+          setDomains: import.meta.env.REACT_APP_MATOMO_DOMAINS
+            ? import.meta.env.REACT_APP_MATOMO_DOMAINS.split(',')
+            : [],
+          setDoNotTrack: undefined,
+          disableCookies: undefined,
+          trackPageView: undefined,
```

Comment:
Since we have a separate trackPageView method in useMatomo hook, this propery can be left out. In this case this actually causes page view to be tracked multiple times.

Reply:
Good find. It actually made page views tracked twice